### PR TITLE
Redesign activity page

### PR DESCRIPTION
**Pull request by Emmanueldmlr**
# Activity Page Redesign

image
Before

image
After

fixes https://github.com/EveripediaNetwork/issues/issues/211
fixes https://github.com/EveripediaNetwork/issues/issues/219

### CODE REVIEW

**Comment by Emmanueldmlr** [comment]
@kesar ,kindly review.

File: src/components/Activity/ActivityCard.tsx [hunk 1]
```diff
@@ -1,164 +1,163 @@
...
 
 interface ActivityCardProps {
   id: string
   title: string
   brief: string
-  editor: string
-  wordsChanged: number
-  isFirstEdit: boolean
-  percentChanged: number
+  editor: string | undefined
```
Comment (kesar) [inline src/components/Activity/ActivityCard.tsx:24]: can we enforce this string again so we dont need to  add the | '' everywhere?

Comment (Emmanueldmlr) [reply src/components/Activity/ActivityCard.tsx:24]: Okay then. I will Implement those changes now. Thanks.

File: src/components/Activity/ActivityCard.tsx [hunk 2]
```diff
@@ -1,164 +1,180 @@
...
-        {isFirstEdit ? 'First Edit ' : `${percentChanged * 100}% Changed `}
-      </Text>
+      <Flex justifyContent="space-between" w="full">
+        <Box>
+          <HStack flex="1">
+            <DisplayAvatar address={wiki.user?.id} size="20" />
+            <Text fontSize="14px" color="linkColor">
+              <NextLink href={`/account/${editor}`} passHref>
+                <Link href="passRef" color="brand.500" fontWeight="bold">
+                  {shortenAccount(editor || '')}
```
Comment (kesar) [inline src/components/Activity/ActivityCard.tsx:77]:
```suggestion
                  {shortenAccount(editor)}
```

**Review by kesar — APPROVED** [review]

**Pull request merged**